cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeDatasetServiceManager.java

```diff
@@ -116,7 +116,7 @@ protected void startUp() throws Exception {
       @Override
       public Boolean call() throws Exception {
         try {
-          getDSFramework().getInstances(Id.Namespace.DEFAULT);
+          getDSFramework().getInstances(NamespaceId.SYSTEM.toId());
```

Comment:
Isn't `DatasetFramework.getInstances` take `NamespaceId`?? Why need to call `toId()`??